lib/execution_engine2/sdk/EE2Status.py

```diff
@@ -474,9 +474,7 @@ def check_jobs(
                 del mongo_rec["_id"]
                 mongo_rec["retry_count"] = len(job["retry_ids"])
                 mongo_rec["job_id"] = str(job.id)
-                mongo_rec["parent_job_id"] = (
```

Comment:
The frontend might still need this.

Reply:
Assuming that this is deleting the `parent_job_id` field from the job object, the FE can get the `parent_job_id` from the `job_input`, so it's fine to remove it from here.

Reply:
Your assumption is correct